src/middlewared/middlewared/plugins/account.py

```diff
@@ -278,7 +278,19 @@ async def query(self, filters, options):
                 }})
 
         if dssearch:
-            return await self.middleware.call('dscache.query', 'USERS', filters, options)
+            dssearch_results = await self.middleware.call('dscache.query', 'USERS', filters, options.copy())
```

Comment:
Why are we doing `options.copy()`?

Reply:
Because we are popping attributes from `options` which is not desirable and can affect the result here in `user.query`